### DISCUSSION TITLE
在學校需不需要用到桌機

### DISCUSSION THREAD

**Discussion by CeBru13** [1101-GS4719-C/Python-Community-GS4719-C#2247] [General]
我自己是只有用筆電，暫時也還沒有一定要到桌機的程度，系上也有係電可以用，但我的朋友有些會帶桌機來學校，可是好像也不一定會用到，桌機還不能像筆電一樣攜帶，這樣的話還有必要帶桌機嗎？

**Comment by gigitilto** [comment]
我認為桌機是一個可有可無的東西，對於喜歡玩遊戲的人來說幾乎是必須的，但對我來說就不必要，一來搬宿舍時需要額外花時間搬運，還得特別小心，二來我也只玩貓咪大戰爭這種簡單小遊戲，用電腦來玩簡直大材小用，再來打報告或讀書有筆電就很足夠了，假如是要跑數據，學校也有電腦可以用，沒必要自己買一台。

**Comment by Chattenoire** [comment]
通常是筆電＋平板就可以解決大部分人的需求，但如果你的科系會需要用到GPU運算或是自己想玩遊戲，那就把筆電換成桌電吧
(當然，你有錢就都買，反正是你的物品)

**Comment by qiu0109** [comment]
桌機通常只是用來打電動的，如果平常有在玩遊戲就可以帶。而且除了部分實習課以外，應該平板就很夠用了，我認為沒必要買桌機。

**Comment by christaaah** [comment]
我覺得大部分的人應該使用筆電就夠了，而且如果是住宿舍的話在搬宿舍的時候會很麻煩，而且在一些程式課就不能用自己的電腦，如果筆電真的不行的話系上也有電腦，但是如果有個人需求例如打遊戲，而且不怕麻煩的話，那買桌機也無所謂。

**Comment by ShuliANN** [comment]
在學校是否需要用到桌機，其實取決於每個人的需求和使用習慣。如果你的需求僅限於日常學習、完成作業或簡單的娛樂，筆電的便攜性和多功能性通常已經足夠。而且，筆電更適合在不同的地方使用，例如教室、圖書館或咖啡廳。

**Comment by ShuliANN** [comment]
而我有些朋友也會帶桌機到學校，可能是因為桌機在某些方面有優勢，例如性能更強，特別適合追劇、玩高效能的遊戲或進行影像編輯等需要大容量運算的活動。不過，桌機缺乏便攜性，搬運和設置都比較麻煩，對於需要經常移動的人來說，確實不是最方便的選擇。

**Comment by irenesunnn** [comment]
有打遊戲或跑模型的需求還是需要桌機
如果只有一般操作，覺得配一個大螢幕就很夠了

**Comment by kzx-creat** [comment]
我有筆電就很夠了，頂多加個平板。但前一陣子有在玩麥塊，那時就覺得如果是用桌機可能會玩得更爽，遊戲跟一些插件會跑得更順。

**Comment by yue083139** [comment]
以我的個人經驗來說桌機不是那麼必要，平常上課打code有筆電就夠了，攜帶也方便，平常打遊戲也是沒有到太高的要求所以還算過得去，配個鍵盤和螢幕也是玩得挺開心，況且也不是每堂課都一定會使用到電腦，為了課程需求的話感覺是沒必要考慮桌機。

**Comment by BOSENCHANG2004** [comment]
以我來說我認為機動性還是最需要的，能夠有一台能夠隨時帶得走的載具能夠做文書或是計算對於大學生還是非常重要的。

**Comment by Kaz9487** [comment]
我的想法是筆電>桌機，不管是報告討論還是專題，能帶著走的載具是比較好的選擇。
當然桌機的性價比絕對比比電高出很多，同樣的價錢桌機能換來更好的性能。但性能跟方便性怎麼權量，那就各憑本事吧

**Comment by a414414** [comment]
我認為桌機其實不是必要，平常使用的話筆電就很足夠了，既方便攜帶又能完成大部分的日常任務。
桌機就只能在一個定點使用，像是家裡，而筆電的話能帶到任何地方，隨時隨地都能使用。
不過也有同學有遊戲的需求，所以就在宿舍或家裡有桌機，那我也是覺得很不錯。

**Comment by XiangLuen** [comment]
對於我來說筆電會更方便，因為桌機無法帶出房間，如果要跟教授meeting也無法，而筆電不管是報告還是寫作業討論都很方便。桌機還有個缺點就是沒供電就會直接關機，想想寫報告寫到一半突然關機肯定很刺激，當然可以用不斷電系統，但我同學跟我分享這個系統也是很耗錢，如果是男十三就更大

**Comment by lichilun** [comment]
我更傾向筆電筆電，因為輕便、靈活，帶到教室、圖書館或咖啡廳都很方便。我覺得筆電的好處就是隨時可以移動，特別是上課做筆記、參加小組討論，或者到自習區寫報告時，用筆電都比桌機方便很多。而且，現在的筆電效能也不差，應付日常作業、查資料或跑一些軟體基本上都夠用。

**Comment by hollowha** [comment]
看人，我筆電用得很開心，因為常常跑咖啡廳或去跟別人討論。

**Comment by jaychang1027** [comment]
我覺得因人而異，當然論方便性來說一定是筆電大於桌機，我自己是覺得可以買螢幕拿來當雙螢幕用就好，其實真的不太需要用到桌機，筆電跟平板就已經很夠用了

**Comment by yuku-huang** [comment]
如果你的用途主要是打報告、查資料、寫程式、修圖修簡報，筆電基本上就夠用了。而且現在很多系都有電腦教室或雲端資源可以用，真的需要跑重一點的程式也不一定非得靠桌機。除非你是念動畫、建模、影片剪輯、機器學習那種需要大量運算跟GPU支援的科系，不然桌機其實不是必要。筆電+外接螢幕就已經能兼顧機動性跟操作效率了。帶桌機來學校反而佔空間又不方便移動，實用性不高。

**Comment by brianou583** [comment]
桌機真的很佔空間，還要螢幕、鍵盤、滑鼠，除非書桌夠大，不然有時反而變負擔。如果筆電性能夠撐，或系上有實驗室可以借用，應該就不用急著買桌機。不過打遊戲、打code就是喜歡在大螢幕、高效能的桌機上進行才會爽，如果有預算的話我可能也會考慮。

**Comment by 12388833** [comment]
桌機的優勢在於同價位下，性能可以比比電高出不少，但筆電的便利性真的贏太多了，不管你要去k中，還是做報告，都是一台筆電、一台平板可以解決的，加上如果你要放桌機的話，基本上宿舍的桌面就小了不少，真的除非你是有遊戲上對性能的追求，不然筆電就很夠用了

**Comment by 77JoyWang77** [comment]
我覺得對於寫程式的資工來說，有個桌機配大一點的螢幕還是不錯的，把資料放onedrive就可以兩邊用了，用習慣了大螢幕，看筆電都會不太習慣呢

**Comment by dandanwu377** [comment]
我覺得在學校不一定需要桌機，大多數情況筆電就足夠應付上課、報告和簡單的程式編寫。除非是念像動畫設計、工程、建模或打算玩遊戲、跑大型模擬的科系，才會比較需要桌機的效能。桌機雖然效能好、價格相對便宜，但不方便帶出門。如果只是文書處理或輕度使用，筆電加外接螢幕也是不錯的選擇。

**Comment by mattwang1222** [comment]
我覺得帶桌機去學校這件事，真的是很見仁見智耶。說實話，桌機體積大又重，根本不像筆電那樣方便隨身攜帶，尤其是在校園裡東奔西跑，背著那種「行動龐然大物」難免累。而且系上又有提供電腦，理論上只要去那邊用就好了，完全沒必要硬拖自己桌機過去。

但我也能理解那些帶桌機去的朋友心態——有時候桌機性能更強，尤其是玩遊戲或跑大型軟體時，筆電難免有性能瓶頸。如果他們偶爾需要用到高性能的工作環境，可能就覺得「還是帶桌機比較安心」。不過，如果只是一般課業需求，其實真的用不到那麼大台的機器。

話說回來，如果真的不太用桌機，反而每天扛著它上下學，還挺累人的，也不划算。要不就乾脆放在家裡當桌機用，需要時回家再用，學校就用系上的設備，這樣比較輕鬆。

**Comment by wuyu06** [comment]
我覺得相比筆電，桌機通常具有更強的硬體規格，適合影像處理、程式開發、影片剪輯或其他需要大量資源的作業。此外，桌機的舒適度與擴充性也比筆電更高，長時間使用時能提供更好的體驗

**Comment by Ting940419** [comment]
我覺得在學校要不要用桌機，其實看人，但就我自己來說，有桌機真的差很多。尤其是資訊相關的課，像寫程式、跑模擬、做專題，有時候筆電跑起來真的會卡，等到你想摔電腦的那種卡。桌機就是性能夠、跑東西比較順，整個人寫程式也會比較有耐心，不會因為電腦慢到不想做。

我那時候就是一開始只有筆電，結果後來遇到一堂要用虛擬機跑Linux的課，筆電直接被操爆，風扇轉到像在起飛，還會過熱關機。後來我自己在宿舍組一台桌機，真的有夠爽，螢幕大又穩定，開虛擬機、寫報告、看教材一條龍，效率高一堆。

當然啦，如果你只是偶爾寫個作業，或者不會在宿舍用電腦，那可能就真的不需要；但如果你跟我一樣常常要寫程式、跑測試、剪影片，有桌機真的會省掉很多麻煩，你就會懂那種差別。

所以如果空間夠、有預算，我自己是蠻推在學校用桌機的，爽度跟效率都差很多。

**Comment by Pyra-0** [comment]
雖然我沒有桌機，但我覺得在學校用不用得到還真不一定。如果說有在打遊戲或者要跑模擬之類的，需要做比較大算力的事，桌機可能就挺必要的，畢竟筆電的能力還是有限的。但如果沒有的話我覺得筆電再可能加個平板就能應付平常生活了，而且桌機機動性相較起來也沒那麼好，誰也不想常常扛著主機到處跑。

**Comment by LINSHIHMIN** [comment]
我自己是沒有在使用桌機，但是有看到朋友在使用，就覺得雙螢幕好像也蠻舒服的，不管是看劇、打遊戲或是做作業都蠻方便的感覺，就是螢幕大大的看起來就比較快樂，但如果對這些沒什麼要求，其實筆記型電腦和平板就做得到了，也不需要特地搬一個桌機來。

**Comment by Piyenparty** [comment]
我覺得桌機對我來說是一個可有可無的東西。對於喜歡玩大型遊戲、追求高效能硬體的人來說，桌機幾乎是必備的，畢竟那種需求不是一般筆電能輕鬆滿足的。但我本身就不是那種重度玩家，我平常玩的遊戲大多也只是像《貓咪大戰爭》這類用手機也可以玩的東西，用一台高規桌機來跑這些遊戲，反而有種大材小用的感覺。

再來就是搬宿舍的問題。桌機又大又重，還有螢幕、鍵盤、主機這些零件，每次搬家都要額外花很多時間和力氣，而且一不小心碰撞還可能損壞，比起來真的很麻煩。相較之下，筆電輕巧方便，平常要打報告、查資料、上課或自修，其實功能就已經很足夠了。

如果真的有跑大型數據、需要高效能運算的時候，學校也有提供電腦教室和資源，不一定需要自己額外購買高階設備。綜合來看，對我這種使用需求不高、活動頻繁、空間有限的人來說，桌機其實並不是一個必要的選擇，筆電反而是更符合我生活型態的工具。
